scripts/_terraform.sh

```diff
@@ -500,8 +500,10 @@ function _get_etl_sibling_aws_account_id() {
   local account=$1
   local tools_account_id=$(_get_tools_account_id)
 
+  local account_name=${account#auth-}
```

Comment:
We'll need to do this otherwise, we'd try to fetch the corresponding secret for `etl-auth-dev` instead of `etl-dev`